starknet_devnet/starknet_wrapper.py

```diff
@@ -177,7 +176,8 @@ async def deploy(self, deploy_transaction: Deploy):
                 tx_hash=tx_hash,
                 status=status,
                 execution_info=execution_info,
-                error_message=error_message
+                error_message=error_message,
+                contract_hash=self.__current_carried_state.contract_states[contract_address].state.contract_hash
```

Comment:
Could you use the `state` variable defined in this function's scope?

Reply:
[7fcfb20](https://github.com/Shard-Labs/starknet-devnet/pull/70/commits/7fcfb20a5406f62d0534c6dbbcdc9a730ccd75d4)